.github/workflows/test_converters.yml

```
@@ -0,0 +1,55 @@
+name: Non Regression Tests - Converters
+
+on:
+  push:
+    branches: [dev]
+  pull_request:
+    branches: [dev]
+
+permissions:
+  contents: read
+
```

Add concurrency group so that to ensure at-most-one execution of this workflow.

```suggestion

concurrency:
  group: '${{ github.workflow }} @ ${{ github.event.pull_request.head.label || github.head_ref || github.ref }}'
  cancel-in-progress: true

```